.travis.yml

```diff
@@ -107,6 +107,10 @@ before_install:
   - git clone --depth=1 --branch=$TRAVIS_BRANCH https://github.com/zonemaster/zonemaster-engine.git
   - ( cd zonemaster-engine && cpanm --verbose --notest . ) && rm -rf zonemaster-engine
 
+    # Install share files
+  - mkdir -p ./lib/auto/share/dist/
+  - ln -s ../../../../share ./lib/auto/share/dist/Zonemaster-Backend
```

Comment:
How does this change relate to this PR?

Reply:
Otherwise the tests fail as `dist_file` cannot find the dist directory for Zonemaster-Backend

Reply:
This PR adds a second path to look for `backend_config.ini`, else Config.pm still falls back to what `dist_file` gives. What has changed that requires a change for Travis? Or is it a bug that you fix at the same time?

Reply:
Before, `dist_file` was not executed as we are using the environment variable, now it is always executed, that is the difference.

Reply:
If I may,  instead of having to install the share files I'd rather avoid executing `dist_file` if the `ZONEMASTER_BACKEND_CONFIG_FILE` variable is set. This would prevent failure such as the Travis one. Would it be possible to execute `dist_file` only if the environment variable is unset?

Reply:
That should be done

Reply:
Nice, thanks. Then the Travis patch could be removed I guess.

Reply:
> Then the Travis patch could be removed I guess.

I would prefer to keep it so that the share directory is correctly installed.

Reply:
All right.

Reply:
> Before, `dist_file` was not executed as we are using the environment variable, now it is always executed, that is the difference.

That change is not included in the description. Please add that.

Reply:
This has been reverted, `dist_file` is now executed only if the variable is not set, as it was before.